PIL/Image.py

```diff
@@ -1049,6 +1108,7 @@ def crop(self, box=None):
         return self._new(self._crop(self.im, box))
 
     def _crop(self, im, box):
+        # type: (Image, LURD) -> Image
```

Comment:
It took me a while to realise what was happening (the code is new to me) but as per my comment below this is clearer now.